guides/common/modules/proc_migrating-ansible-content.adoc

```diff
@@ -0,0 +1,34 @@
+[id="migrating-ansible-content_{context}"]
+= Migrating Ansible Content
+
+The upgrade from {RHEL} 7 to {RHEL} 8 includes an upgrade of Ansible core from 2.9 to 2.12.
+
+If you have custom Ansible content such as Playbooks, Job Templates inside REX, Roles and Collections on disk, and you relied on modules being delivered by the Ansible RPM on {project}, you have to consider that you have to migrate your Ansible content.
```

Comment:
Upstream never provided Ansible, it always pointed to some other repository. Overall this looks like a very downstream focused procedure. Perhaps keep this in Satellite only (by guarding it in `master.adoc`)?